std_msgs/Header header
sensor_msgs/Image image
int16 coordX
int16 coordY
int16 coordZ
